client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/persistence/KaaClientState.java

```diff
@@ -31,6 +32,11 @@
     boolean isRegistered();
     void setRegistered(boolean registered);
 
+    /**
+     * User can choose to create key pair or not. In default case there is no key generation
+     *
```

Comment:
Description of key generation strategies should be for <b>isUserKeyStrategy</b> parameter. The methods below don't have any arguments and just return keys. Description specifying what kind of key is returned should be added individually to each method.